src/env/api/client.js

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+
+const client = {
+	BASE_URL: "https://african--market.herokuapp.com/api/",
+}
+client.axios = () => {
+	return axios.create({
+		baseURL: "https://african--market.herokuapp.com/api/",
+	});
+};
+
+const getProductById = async (id) => {
+	const res = await client.axios().get(`products/${id}`);
```

Comment:
Will probably not work as expected per backend documentation (discussed directly via zoom)